neurokit2/ecg/ecg_findpeaks.py

```diff
@@ -935,7 +935,7 @@ def _ecg_findpeaks_WT(signal, sampling_rate=1000):
 # =============================================================================
 
 
-def _ecg_findpeaks_rodrigues(signal, sampling_rate=1000):
+def ecg_findpeaks_rodrigues(signal, sampling_rate=1000):
```

Comment:
```suggestion
def _ecg_findpeaks_rodrigues(signal, sampling_rate=1000):
```

This needs to remain an internal function since they are called by the main `ecg_findpeaks` :)